data/tutorials/lg_08_error_handling.md

```diff
@@ -196,6 +888,57 @@ match Sys.os_type with
+```
+Here, it wouldn't be correct to use `failwith` because it requires a corrupted
+system or the compiler to be bugged for the second code path to be executed.
+Breakage of the language semantics qualifies as extraordinary circumstances. It
+is catastrophic!
+
+# Concluding Remarks
+
+Properly handling errors is a complex matter. It is [cross-cutting
+concern](https://en.wikipedia.org/wiki/Cross-cutting_concern), touches all parts
+of an application, and can't be isolated in a dedicated module. In contrast to
+several other mainstream languages, OCaml provides several mechanisms to handle
+exceptional events, all with good runtime performance and code
+understandability. Using them properly requires some initial learning and
+practice. Later, it always requires some thinking, which is good since proper
+error management shouldn't ever be overlooked. No error handling is better
+than the others, and is should be matter of adequacy to the context rather than
+of taste. But opinionated OCaml code is also fine, so it's a balance.
+
+# External Ressources
+
+- [“Exceptions”](https://v2.ocaml.org/releases/5.0/htmlman/coreexamples.html#s%3Aexceptions) in ”The OCaml Manual, The Core Language”, chapter 1, section 6, December 2022
+- [Module **`option`**](https://v2.ocaml.org/releases/5.0/api/Option.html) in OCaml Library
+- [Module **`result`**](https://v2.ocaml.org/releases/5.0/api/Result.html) in Ocaml Library
+- [“Error Handling”](https://dev.realworldocaml.org/error-handling.html) in “Real World OCaml”, part 7, Yaron Minsky and Anil Madhavapeddy, 2ⁿᵈ edition, Cambridge University Press, October 2022
+- “Add "finally" function to Pervasives”, Marcello Seri, GitHub PR, [ocaml/ocaml/pull/1855](https://github.com/ocaml/ocaml/pull/1855)
+- “A guide to recover from interrupts”, Guillaume Munch-Maccagnoni, parf the [`memprof-limits`](https://gitlab.com/gadmm/memprof-limits/) documentation
+
+# Acknowledgements
+
+- Authors
+  1. Simon Cruanes [@c-cube](https://github.com/c-cubeauthored)
+  2. John Whitington [@johnwhitington](https://github.com/johnwhitington)
+  3. Cuihtlauac Alvarado [@cuihtlauac](https://github.com/cuihtlauac)
+- Contributors
+  * Dan Frumin [@co-dan](https://github.com/co-dan)
+  * Jean-Pierre Rodi
+  * Thibaut Mattio [@tmattio](https://github.com/tmattio)
+  * Jonah Beckford [@jonahbeckford](https://github.com/jonahbeckford)
+- Suggestions and Corrections:
+  * Claude Jager-Rubinson
+  * [@rand00](https://github.com/rand00)
+  * Guillaume Munch-Maccagnoni [@gadmm](https://github.com/gadmm)
+  * Edwin Török [@edwintorok](https://github.com/edwintorok)
+  * Kim Nguyễn [@Tchou](https://github.com/Tchou)
+  * Ashine Foster [@AshineFoster](https://github.com/AshineFoster)
+  * Miod Vallat [@dustanddreams](https://github.com/dustanddreams)
+  * Christine Rose [@christinerose](https://github.com/christinerose)
+  * Riku Silvola [@rikusilvola](https://github.com/rikusilvola)
+  * Guillaume Petiot [@gpetiot](https://github.com/gpetiot)
```

Comment:
```suggestion
<!--
Acknowledgements

- Authors
  1. Simon Cruanes [@c-cube](https://github.com/c-cubeauthored)
  2. John Whitington [@johnwhitington](https://github.com/johnwhitington)
  3. Cuihtlauac Alvarado [@cuihtlauac](https://github.com/cuihtlauac)
- Contributors
  * Dan Frumin [@co-dan](https://github.com/co-dan)
  * Jean-Pierre Rodi
  * Thibaut Mattio [@tmattio](https://github.com/tmattio)
  * Jonah Beckford [@jonahbeckford](https://github.com/jonahbeckford)
- Suggestions and Corrections:
  * Claude Jager-Rubinson
  * [@rand00](https://github.com/rand00)
  * Guillaume Munch-Maccagnoni [@gadmm](https://github.com/gadmm)
  * Edwin Török [@edwintorok](https://github.com/edwintorok)
  * Kim Nguyễn [@Tchou](https://github.com/Tchou)
  * Ashine Foster [@AshineFoster](https://github.com/AshineFoster)
  * Miod Vallat [@dustanddreams](https://github.com/dustanddreams)
  * Christine Rose [@christinerose](https://github.com/christinerose)
  * Riku Silvola [@rikusilvola](https://github.com/rikusilvola)
  * Guillaume Petiot [@gpetiot](https://github.com/gpetiot)
-->
```

Can we do the acknowledgements in a comment?

It's great to acknowledge people's efforts, but over time this grows very large and distracts from the content itself. In the sequences documentation, the same section is called "Credits". :thinking: